packages/suite/src/views/wallet/send/components/Header/index.tsx

```diff
@@ -29,6 +29,14 @@ const HeaderRight = styled.div`
     flex: 1;
 `;
 
+const StyledTitle = styled.h2`
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    text-transform: capitalize;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    font-size: ${variables.FONT_SIZE.H2};
+    padding-top: 10px;
+`;
+
```

Comment:
There are some styles set in HeaderLeft and you override them here. However, HeaderLeft contains only StyledTitle. Can you somehow merge it?

Reply:
a7f45bee6cccace4053ec3c69b5463e881333cf4